### DISCUSSION TITLE
isActive() doesn't work properly

### DISCUSSION THREAD

**Discussion by FlTr** [datenstrom/community#77] [Report a bug] [answered]
Content structure:

```
content/
  - stuff/
    - page.txt
    - file.txt
    - file_v2.txt
```

If we go to _/stuff/file_ all is ok, but if we go to _/stuff/file_v2_, file and file_v2 are marked as active.

**Comment by markseuffert** [comment, marked as answer]
Sorry about that. Should be fixed now.

**Comment by FlTr** [comment]
will test it soon, thanks

**Comment by FlTr** [comment]
It works, could be closed.
